packages/preview/g-exam/0.2.0/template/main.typ

```diff
@@ -0,0 +1,36 @@
+#import "@preview/g-exam:0.2.0": g-exam, g-question, g-subquestion
+
+#show: g-exam.with(
+  school: (
+    name: "My School",
+    logo: read("./logo.png", encoding: none),
```

Comment:
I would use `image("logo.png")` rathen than `read` here. To affects it sizing, you can then do the following in your template:
```
set image(width: 2.5cm, fit: "contain")
logo
```

Reply:
I was considering using image, but I thought it was better to put as little dimension as possible in the exam document. I would like this to be used by unenthusiastic teachers of markup languages. You have to make it very easy to use, that's why I'm interested in sugar syntax.

I'm not going to use the image set, because it changes the size of all the images, I've changed read to image in the home document.

I don't know what you think, but in the next version, I'm going to put several examples inside the screen, as an example. The way I work is to create exams in the same folder. It is not the same to create a thesis, in which you work on it over a long period of time, than to work on creating exams, in which you create several in a month.

Reply:
> Can I set a maximum width for a piece of content?

You can use a `block` to force a specific width. Maximum width is trickier, though probably possible with `measure`.